test/integration/parallel_worker_test.js

```diff
@@ -15,12 +15,11 @@ suite('Parallel workers', function() {
   var PARALLEL_TOTAL = 2;
 
   var workerA, workerB;
-  var workerType = `test_worker_${slugid.v4()}`.substring(0,22);
 
   setup(co(function * () {
     // Each worker should use the same worker type but a unique worker id.
-    workerA = new TestWorker(DockerWorker, workerType, slugid.v4());
-    workerB = new TestWorker(DockerWorker, workerType, slugid.v4());
+    workerA = new TestWorker(DockerWorker, 'dummy-parallel-type');
+    workerB = new TestWorker(DockerWorker, 'dummy-parallel-type');
```

Comment:
Ah yea, this needs to be fixed.  This was done after Jonas and I talked about having wokertype static for the tests because of azure queues, but then I realized it was a mistake and reverted the changes I made in testworker.js but not here.  thanks for the catch.